client/src/Tickets/Tickets.jsx

```diff
@@ -103,15 +87,16 @@ const Tickets = () => {
         });
         setData(data);
       });
-    if (user) {
-      setTicket((prevTicket) => ({ ...prevTicket, UserId: user.id }));
-    }
-  }, [setData, user]);
+  }, []);
 
-  function updateTable(ticket) {
+  function onCreate(ticket) {
     setData([...data, ticket]);
```

Comment:
Since this component is the one managing the data, this is the component that knows how to modify its data on changes.